ota-plus-web/app/com/advancedtelematic/api/ApiClient.scala

```diff
@@ -48,6 +50,20 @@ object ApiRequest {
   }
 }
 
+trait CirceJsonBodyWritables {
+
+  implicit def circeJsonBodyWritable: BodyWritable[io.circe.Json] =
+    BodyWritable(json => InMemoryBody(ByteString.fromString(json.noSpaces)), "application/json")
```

Comment:
This wasn't needed in other API calls?

Reply:
The other calls use the Play JSON library, I wanted to use Circe because there is a KeyType encoder already.